### DISCUSSION TITLE
Does `app.addy.io/failed-deliveries` include both outbound (delivery) and inbound (receipt) failures?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#701] [Q&A] [answered]
At [`app.addy.io/failed-deliveries`](https://app.addy.io/failed-deliveries), I have a failure notice for a message which appears to be from myself to an address, rather than a response by them:

![screenshot-app addy io-2024 11 27-21_14_46](https://github.com/user-attachments/assets/41323280-3f3e-443f-bbdf-56ab8e69c9af)

What confuses me is that the failure notice has appeared approximately 4 days after I initially sent the message.

If [`app.addy.io/failed-deliveries`](https://app.addy.io/failed-deliveries) solely contains outbound/delivery failures, this means I'll contact [`support@protonmail.zendesk.com`](mailto:support@protonmail.zendesk.com) instead.

**Comment by willbrowningme** [comment, marked as answer]
The message was deferred with a soft bounce (4.X.X) because the addy.io mail server could not connect to their mail server. It was automatically retried by Postfix a number of times over a few days.

After so many retries had failed, the message was marked as undeliverable which is why it only just showed up in your failed deliveries.

It doesn't even look like that email you tried to send to has MX records so that's why it can't receive any emails - https://www.hardenize.com/report/volunteer.sja.org.uk/1732801832#email

**Reply by roke-julian-lockhart** [reply]
> It doesn't even look like that email you tried to send to has MX records so that's why it can't receive any emails

@willbrowningme, thanks. The SJA's technical incompetence is mind-boggling. I'll forward this to them too.
